connection.go

```diff
@@ -735,6 +735,10 @@ func buildSnowflakeConn(ctx context.Context, config Config) (*snowflakeConn, err
 		queryContextCache:   (&queryContextCache{}).init(),
 		currentTimeProvider: defaultTimeProvider,
 	}
+	err := initEasyLogging(config.ClientConfigFile)
```

Comment:
Should we init it on every new connection? Maybe on initializing driver is good enough?

Reply:
we don't init on every new connection, for most of the cases we will immediately return from this method without taking any further actions. We really init only for the first time and for the first time when client config file is given explicite. It is not enough to do it on driver's initalization because it can be reconfigured based on value given in config parameter.